tools/setup_helpers/generate_code.py

```diff
@@ -66,13 +69,20 @@ def generate_code(ninja_global=None,
             selected_op_list=selected_op_list,
             force_schema_registration=force_schema_registration)
 
+    if subset == "python" or not subset:
+        gen_annotated(
+            declarations_path or DECLARATIONS_PATH,
+            python_install_dir,
+            autograd_dir)
+
 
 def main():
     parser = argparse.ArgumentParser(description='Autogenerate code')
     parser.add_argument('--declarations-path')
     parser.add_argument('--nn-path')
     parser.add_argument('--ninja-global')
     parser.add_argument('--install_dir')
+    parser.add_argument('--python_install_dir')
```

Comment:
This is probably going to need fbcode specific changes (no action needed from you)